scripts/omg.yml

```
@@ -27,4 +27,5 @@ runcmd:
     admin_console_install: False
     admin_console_enabled: False
     EOF
-  - curl iiab.io/risky.txt | bash &
+  - curl iiab.io/risky.txt | bash
+  - curl https://raw.githubusercontent.com/iiab/calibre-web/master/scripts/calibre-web-nginx.conf > /etc/nginx/conf.d/calibre-web-nginx.conf
```

If PR #50's 3 Ansible vars are temporarily hard-coded to {`/books`, `/libros`, `/libros`} during testing (?) then perhaps consider...?

```suggestion
  - curl https://raw.githubusercontent.com/iiab/calibre-web/master/scripts/calibre-web-nginx.conf > /etc/nginx/conf.d/calibre-web-nginx.conf
  - nginx -s reload
```